com.unity.render-pipelines.high-definition/Documentation~/Render-Pipeline-Wizard.md

```diff
@@ -8,16 +8,15 @@ To open the **Render Pipeline Wizard**, go to **Window > Render Pipeline** and s
 
 ## Packages
 
-At the top of the window, there is an information box that shows you the currently installed version of HDRP, as well as the latest version of HDRP that is compatible with your current Unity version.
+At the top of the window, there is an information text that shows you the currently installed version of HDRP. The button **Check update** provide you a quick shortcut to the **Package Manager** windows onto the HDRP package.
```

Comment:
The **Check Update** button provides a shortcut to the HDRP window in the Package Manager.

Reply:
Well, not sure but now I understand that HDRP have a window but it is in fact Package Manager windows, with HDRP package selected in the package list on the left, so the HDRP package description is shown on the right.

Perhaps this?
The **Check Update** button provides a shortcut to the HDRP package in the Package Manager window.

Reply:
Yeah, sounds good